XamlControlsGallery/Navigation/NavigationRootPage.xaml.cs

```diff
@@ -167,11 +167,10 @@ private void SetDeviceFamily()
 
         private void OnNewControlsMenuItemLoaded(object sender, RoutedEventArgs e)
         {
-            if (IsFocusSupported)
+            if (IsFocusSupported && NavigationViewControl.IsPaneOpen)
```

Comment:
Instead of checking for whether or not the Pane is open, I recommend checking whether the DisplayMode is Expanded.

Reply:
Fixed that in the newest commit.